CMakeLists.txt

```diff
@@ -40,9 +40,25 @@ include(${TORCH_XPU_OPS_ROOT}/cmake/ONEMKL.cmake)
 include(${TORCH_XPU_OPS_ROOT}/cmake/BuildFlags.cmake)
 
 option(USE_XCCL "Build with XCCL support" OFF)
+set(USE_C10D_XCCL OFF CACHE BOOL "Build with XCCL support for C10D")
 
-if(NOT WIN32 AND USE_XCCL)
+if(NOT DEFINED USE_XCCL)
+  # Propagate the option to PyTorch
+  caffe2_update_option(USE_XCCL OFF)
```

Comment:
Please be aware that keeping `USE_XCCL` in `torch-xpu-ops` is a short-term implementation and we will upstream this option to stock PyTorch (should happen in PyTorch 2.8 release).